### PR TITLE
bug 1241176:Be less red

### PR DESCRIPTION
**Pull request by techchic**
make task_inspector <code> less "DANGER" colored.

### CODE REVIEW

**Comment by jonasfj** [comment]
I think there is a `format.less` that imported everywhere through layout.jsx, maybe that's is a better place to do this...

Maybe we could keep the background grey and just make the text black so it's still distinguishable from normal text.

**Comment by djmitche** [comment]
Agreed.  I think this is the right CSS, but it needs to apply to all pages in the site.

**Comment by jonasfj** [comment]
Assign @djmitche to this as I shouldn't be arguing color scheme :)

File: task-inspector/taskinspector.less [hunk 1]
```diff
@@ -1 +1,5 @@
 @import "../lib/ui/taskview.less";
+code {
+	color: #2C3638 ;
+	background-color: #FFF ;
```
Comment (walac) [inline task-inspector/taskinspector.less]: nit: coding format should be 2 spaces.

**Comment by techchic** [comment]
I have transferred the code to format.less

**Comment by djmitche** [comment]
You also omitted the `background-color: #FFF`, but I think that's what @jonasfj wanted, and I like the look overall:

screenshot from 2016-03-07 17-36-33

**Comment by djmitche** [comment]
Thanks @techchic!

**Pull request merged**